File: Python/q03.py
num2 = 0

# Comando de repetição while utilizando else

while num2 == 0:
    num1 = float(input('Informe um número: '))
    num2 = float(input('Digite ou outro número: '))
    
    if num2 == 0:
      print('===============================\nDivisor igual a 0 inválido.\nDigite Novamente!\n===============================')
else:
    print(f'O resultado da divisão de {num1} por {num2} é igual a {num1 / num2}.')
    
# Comando de repetição while simples, com validação utilizando if/else               
                     
# while num2 == 0:
#     num1 = float(input('Informe um número: '))
#     num2 = float(input('Digite ou outro número: '))
    
#     if num2 != 0:
#         print(f'O resultado da divisão de {num1} por {num2} é igual a {num1 / num2}.')
#     else:
#         print('===============================\nDivisor igual a 0 inválido.\nDigite Novamente!\n===============================')
        

File: Java/Q03.java
import java.util.Scanner;

public class Q03 {
    public static void main(String[] args) {
        Scanner dado = new Scanner(System.in);

        System.out.println("Informe um número: ");
        int num1 = dado.nextInt();
        System.out.println("Informe outro número: ");
        int num2 = dado.nextInt();

        System.out.printf("%s%d%s%d%s%.2f", "O resultado da divisão de ", num1, " por ", num2, " é igual a ", (float) num1 / num2);
    }
}
